### PR TITLE
Docs: Added privacy policy page

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Description

Added privacy policy to Website instead of referencing to .github/PRIVACY.md

Because of this, @yaira2 has to update the link of this to reference to website's link.

## Screenshots (if appropriate):

IMG_0308

### CODE REVIEW

**Comment by yaira2** [comment]
I'm wondering if this would be better as a standalone page.

**Comment by 0x5bfa** [comment]
Not in docs page?
I agree ☝️

File: src/layout/Footer/Footer.svelte [hunk 1]
```diff
@@ -117,6 +117,12 @@
 			</Button>
 		{/each}
 	</div>
+	<div class="column">
+		<p>Support</p>
+		<Button variant="hyperlink" href="/privacy">
+			Privacy policy
```
Comment (yaira2) [inline src/layout/Footer/Footer.svelte:123]:
```suggestion
			Privacy
```
Can you also localize the text?

File: src/routes/privacy/+page.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<main class="root">
+
+# Privacy policy
+
+Effective date: September 4th, 2024
+
+This Privacy Policy ("Policy") for Files ("we", "us", or "our") describes how and why we collect, store, use, and disclose information about users ("you", "your") when you use our services ("Services"), and our desktop application (the "Application", "Files"). By using our Services, you acknowledge and consent to the practices described in this Policy.
+
+## Table of Contents
```
Comment (yaira2) [inline src/routes/privacy/+page.md:9]: If we decide to add a table of contents, we could generate it from the headers and make it scroll to the relevant sections. In the meantime, let’s remove this section.

File: src/routes/privacy/+page.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Information We Collect
+
+**Diagnostic Information.** Our Application collects non-personably identifiable diagnostic data such as fault analysis and performance logs. This information may include technical details about your device, such as its make, model, operating system, and Application version. In addition, Files may store log files containing diagnostic information on the user's device. These files are not shared with us by default.
+
+**Usage Information.** We collect non-identifiable information about your use of our Services, including but not limited to interactions within the Application, enabled user preferences (Application settings), and usage patterns. Our Application uses Sentry for error reporting, please review the Sentry Privacy Policy here: https://sentry.io/privacy/.
+
+Furthermore, we use certain Microsoft services such as Microsoft Partner Center to collect usage data, to learn more about how data is collected, used, and disclosed by Microsoft and its subsidiaries, please review the Microsoft Privacy Policy Statement available here: https://privacy.microsoft.com/privacystatement
```
Comment (yaira2) [inline src/routes/privacy/+page.md:15]:
```suggestion
Furthermore, we may use certain Microsoft services such as Microsoft Partner Center to collect usage data, to learn more about how data is collected, used, and disclosed by Microsoft and its subsidiaries, please review the Microsoft Privacy Policy Statement available here: https://privacy.microsoft.com/privacystatement
```

**Comment by yaira2** [comment]
@0x5bfa what does the updated footer look like?

**Comment by 0x5bfa** [comment]
image

**Review by Lamparter — APPROVED** [review]
What's stopping this from being merged?

**Comment by yaira2** [comment]
Work needs to be done on the design.

**Comment by yaira2** [comment]
I'm going to merge into a separate branch so I can make some more changes.

**Pull request merged**